tutorial/chapter/4/index.md

```diff
@@ -50,46 +42,27 @@ not 4
 
 ## Comparison operators
 
-The operators `<`, `<=`, `>`, and `>=` work like JavaScript, except they throw
-an error unless both values are strings or both values are numbers. It doesn't
-make much sense to ask if an array or an object is less than something else, so
-this helps avoid subtle bugs.
-
-The operator `==` is a deep equality operator. It works like JavaScript `===`
-for the purpose of comparising strings, numbers, `true`, `false`, `undefined`,
-`null`, and functions, but differs for arrays, and objects.
+The operators `<`, `<=`, `>`, and `>=` work like JavaScript, except they throw an error unless both values are strings or both values are numbers. It doesn't make much sense to ask if an array or an object is less than something else, so this helps avoid subtle bugs.
 
-For arrays, it checks to see if all elements of both arrays are equal to each
-other based on the Squiggle `==` operator. It is not aware of sparse arrays.
-
-For objects, it checks all keys returned by `Object.keys` on both objects for
-equality using the Squiggle `==` operator.
+The operator `==` is a strict equality operator. It works like JavaScript `===` for the purpose of comparing strings, numbers, `true`, `false`, `undefined`, and `null`, but it throws an exception if you try to compare objects, arrays, or functions. This is because those types do not have well-defined value comparison, so you should implement it yourself.
```

Comment:
lodash can help, and I might add my own into the mix. Perhaps I should add "because your interpretation of value comparison may differ from mine here"?